docs/source/index.rst

```diff
@@ -67,25 +67,34 @@ Supported datasets
    :caption: Using docTR
    :hidden:
 
-   using_models
-   using_model_export
+   models/using_models
+   models/using_model_export
```

Comment:
I disagree, in the future, this will hold tutorials / pages about multiple aspects (not only models), so let's put it in `using_doctr`